source/renderer/app/api/assets/types.js

```diff
@@ -1,77 +1,88 @@
 // @flow
 import BigNumber from 'bignumber.js';
 
-import Asset from '../../domains/Asset';
+import AssetDomain from '../../domains/Asset';
 
-export type SingleAsset = {
+/**
+ *
+ * ASSET
+ * Fetched from the Assets API endpoint
+ * It's not attached to a particular wallet or transaction
+ * Therefore, it doesn't have `quantity` nor `address`
+ *
+ * Exclusive data: fingerprint, metadata
+ * Missing data: quantity, address
+ *
+ */
+export type ApiAssets = Array<ApiAsset>;
```

Comment:
Let's move the array type below the single type it is referencing (= move `ApiAssets` definition below the `ApiAsset` definition)

Reply:
✔️